package/autoyast2.changes

```diff
@@ -1,3 +1,12 @@
+-------------------------------------------------------------------
+Fri Nov 12 13:21:54 UTC 2021 - Imobach Gonzalez Sosa <igonzalezsosa@suse.com>
+
+- bsc#1192626:
+  - Registration takes place always at the same point, no matter
+    which installation medium is being used.
+  - Adapt to the product specification API.
```

Comment:
changelog should include also fix for missing product bug - https://github.com/yast/yast-autoinstallation/blob/SLE-15-SP3/package/autoyast2.changes#L4